test/webserv/integration/test_http_post.py

```
@@ -0,0 +1,137 @@
+
+def delete_file(file_path):
+    if file_path is None:
+        return
+
+    try:
+        if os.path.exists(file_path):
+            os.remove(file_path)
+            print(f"Deleted file: {file_path}")
+    except Exception as e:
+        print(f"Error deleting file: {file_path}, {e}")
+        assert False
```

Ruff

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)